docs/api/effector/Effect.md

```diff
@@ -32,20 +48,59 @@ The only requirement for function:
 
 ## Effect Methods
 
-### <a id='use'></a>[`use(fn)`](#use)
+### <a id='use'></a>[`use(thunk)`](#use)
+
+Setup function to call when effect trigger.
+
+Replaces previous function inside.
+
+#### Arguments
+
+(_`thunk`_): Function, that receive arguments passed to effect call
 
 #### Returns
 
 (_`Effect`_): A container for async function.
```

Comment:
It returns promise

Reply:
`createEffect().use(fn)` returns promise??